shell/circleci/setup.sh

```diff
@@ -31,6 +31,13 @@ if [[ -n $TEST_RESULTS ]]; then
   mkdir -p "$TEST_RESULTS"
 fi
 
+# run prescript if user specified to install packages etc. before tests
+if [[ -n $PRETEST_SCRIPT ]]; then
+  echo "⚙️ running prescript file ${PRETEST_SCRIPT}"
```

Comment:
nit: match case of other statements

```suggestion
  echo "⚙️ Running setup script \"${PRETEST_SCRIPT}\" (from PRETEST_SCRIPT)"
```